roles/consul/handlers/main.yml

```diff
@@ -3,3 +3,9 @@
     name: consul
     enabled: yes
     state: restarted
+
+- name: restart dnsmasq
+  service:
+    name: dnsmasq
+    enabled: yes
```

Comment:
No need to enable this service here. Let's be more explicitly and just restart ``dnsmasq``.